test/conformance/testing/CMakeLists.txt

```diff
@@ -4,7 +4,11 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 add_ur_library(ur_testing STATIC 
-    source/utils.cpp)
+    source/utils.cpp
+    source/fixtures.cpp)
 target_include_directories(ur_testing PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
-target_link_libraries(ur_testing PRIVATE gtest_main unified-runtime::headers)
+target_link_libraries(ur_testing PRIVATE
+    gtest_main
+    ${PROJECT_NAME}::common
```

Comment:
Just a question as I am a little confused by this, but which common files should this link?

Reply:
it's because fixtures.hpp includes [checks.h](https://github.com/oneapi-src/unified-runtime/blob/main/test/conformance/testing/include/uur/checks.h#L11) which includes [ur_params.hpp](https://github.com/oneapi-src/unified-runtime/blob/main/source/common/ur_params.hpp). Previously fixtures.hpp wasn't actually included in the `ur_testing` target, it isn't used in utils.cpp

Reply:
Ah, that make sense, Thanks!